mode/vue/index.html

```diff
@@ -0,0 +1,88 @@
+<!doctype html>
```

Comment:
@marijnh the question is just if .vue files will be highlighted correctly. But... After your comment I started to think that vue mode should be just some wrapper over a refactored htmlmixed. What do you think?